TipulDay.py

```diff
@@ -1,178 +1,182 @@
     response = requests.get(url)
-    
-        # parse the HTML
+
+    # parse the HTML
     soup = BeautifulSoup(response.text, 'html.parser')
 
-        # get the top 1000 websites from the search results
+    # get the top 1000 websites from the search results
     for link in soup.find_all('a'):
         if link.get('href').startswith('/url?q='):
             websites.append(link.get('href')[7:])
-    
-    SearchStartNum=SearchStartNum+10 #increases after every loop to start in +10 search value
-    url=f'https://www.google.com/search?q={SearchString}&start={SearchStartNum}'
 
-print("Finished Collecting Website Data.")
+    # increases after every loop to start in +10 search value
+    SearchStartNum = SearchStartNum+10
 
-###############################################################################################################################################################################
+print("[*] Websites collected")
 
-## Code that inputs the form into all the websites
+###############################################################################################################################################################################
 
-print("Started Submitting Forms.")
+# Code that inputs the form into all the websites
 
-if(len(websites)>0):
+print("[*] Submitting forms")
 
-    SuccessfullForms=0
-    FailedForms=0
+if(len(websites) > 0):
+    SuccessfullForms = 0
+    FailedForms = 0
     for website in websites:
-        #specifying the website url
+        # specifying the website url
         url = website
 
-        #making a request to the website and getting the response 
+        # making a request to the website and getting the response
         try:
             response = requests.get(url)
         except:
-            print("Error 5: Issue requesting from the website.")
-            FailedForms=FailedForms+1
+            print("[!] Error 5: Issue requesting from the website")
+            FailedForms += 1
             continue
 
-        #parsing the response using BeautifulSoup
+        # parsing the response using BeautifulSoup
         soup = BeautifulSoup(response.text, 'html.parser')
 
-        #finding the form elements in the page 
+        # finding the form elements in the page
         form_elements = soup.find_all('input')
         #form_elements.append(soup.find_all('input', {'type': 'tel'}))
-        #creating a dictionary to store the form data
+        # creating a dictionary to store the form data
         form_data = {}
 
-        #looping through the form elements and getting the data 
+        # looping through the form elements and getting the data
         for element in form_elements:
-                name = element.get('name')
-                value = element.get('value')
-                form_data[name] = value
+            name = element.get('name')
+            value = element.get('value')  # why is the value necessary?
```

Comment:
why do you think its not?

Reply:
Because you use your own values when you are sending the form, in addition I havent seen you using the gathered values anywhere else